src/components/SubmitButton.jsx

```diff
@@ -0,0 +1,19 @@
+export function SubmitButton() {
+
+  return (
+    <>
+      <button type="submit"
+        onMouseOver={e => {
```

Comment:
The Element.classList has a [toggle](https://devdocs.io/dom/element/classlist) method that you can use. With this you can create a single function that adds or remove className.

Looking at this, I think the code loses its readability. Not too sure if I like this. Oh well at least you now know about toggle (assuming you did not know about it).
```
function updateBackground(event) {
    event.target.classList.toggle("bg-cyan-400")
    event.target.classList.toggle("bg-cyan-700")
}
```